config/routes.rb

```diff
@@ -11,7 +11,7 @@
   get "/media/:id/:filename" => "media#download", :constraints => { filename: /.*/ }
   get "/government/uploads/*path" => "whitehall_media#download"
 
-  if Rails.env.development?
+  if Rails.env.development? || ENV["USE_FAKE_S3"]
```

Comment:
I think it would be better to extract the concept of using the fake S3 mechanism into a single place to reduce duplication and make the code easier to understand.

Reply:
@adrianclay FYI I'm about to extract this concept for other reasons as a separate PR.

Reply:
I've opened https://github.com/alphagov/asset-manager/pull/349 to address this concern.